Backend/app/rider-platform/rider-app/Main/src/Domain/Action/UI/Maps.hs

```diff
@@ -27,6 +27,7 @@ where
 
 import qualified Data.Geohash as DG
 import Data.Text (pack)
+import qualified Domain.Action.SimulatedFlow.Maps as SF
```

Comment:
Domain.Action... modules should never import each other (except when you wanna factor out bunch of code from main logic into deeper module, i.e something like Domain.Action.{..}.Internals). Anyway in this case i think it would be better to store it somewhere within SharedLogic.SimulatedFlow.Maps